com.spotify.Client.json

```diff
@@ -105,6 +105,21 @@
                 }
             ]
         },
+        {
+            "name": "spotifywm",
+            "buildsystem": "simple",
+            "build-commands": [
+                "make",
+                "install -Dm644 spotifywm.so /app/lib/spotifywm.so"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/dasJ/spotifywm/archive/master.zip",
```

Comment:
`master` is constantly changing. Pick a specific commit. Probably use the `git` source type.

Reply:
Well, *this* `master` isn't constantly changing, it hasn't been touched in four years. (Which is a frightening testament to how much Spotify cares about its Linux client; four years and it's still needed.) That said though, I am definitely on board with doing the right thing for the sake of enforcing good practices and covering our butts in case the repo suddenly does update down the road. :)